File: LeetCode/Simulation/Python/Leetcode1773-count-items-matching-a-rule.py
class Solution:
    def countMatches(self, items: List[List[str]], ruleKey: str, ruleValue: str) -> int:
        ans, key, n = 0, 0, len(items)
        if ruleKey == "color": key = 1
        elif ruleKey == "name": key = 2
        for i in range(n):
            if items[i][key] == ruleValue: ans += 1
        return ans

File: LeetCode/Simulation/Rust/Leetcode1773-count-items-matching-a-rule.rs
impl Solution {
  pub fn count_matches(items: Vec<Vec<String>>, rule_key: String, rule_value: String) -> i32 {
      let (n, mut ans, mut key) = (items.len(), 0, 0);
      if &rule_key == "color" {
          key = 1;
      } else if &rule_key == "name" {
          key = 2;
      }
      for i in 0..n {
          if &items[i][key] == &rule_value {
              ans += 1;
          }
      }
      ans
  }
}